stubs/SQLAlchemy/sqlalchemy/testing/engines.pyi

```diff
@@ -37,14 +43,22 @@ class ReconnectFixture:
     def restart(self) -> None: ...
 
 def reconnecting_engine(url: Any | None = ..., options: Any | None = ...): ...
+@overload
+def testing_engine(  # type: ignore[misc]
+    url: URL | None = ...,
```

Comment:
It does `url = make_url(url)`, so passing in a string instead of `URL` instance will work too.